hummingbot/templates/conf_pure_market_making_strategy_TEMPLATE.yml

```diff
@@ -108,3 +108,11 @@ take_if_crossed: null
 
 # For more detailed information, see:
 # https://docs.hummingbot.io/strategies/pure-market-making/#configuration-parameters
+
+# Use user provided orders to directly override the orders placed by order_amount and order_level_parameter
+# This is an advanced feature and user is expected to directly edit this field in config file
+# Below is an sample input, the format is a dictionary, the key is user-defined order name, the value is a list which includes buy/sell, order level, and order amount
+# order_override:
+#   override_order_one:["buy","0.5","100"]
```

Comment:
Can user just input [buy, 0.5, 100] (without quotes) here? That'll be more inline with how they input values for other parameters.

Reply:
the keys for `order_override` are bit too long, i think we can just use `order_one`, `order_two`,...

Reply:
Yes, users can use any keys, it's just a string, doesn't matter, I will change to order_one in the example